packages/controllers/profile_edit.coffee

```diff
@@ -0,0 +1,32 @@
+if Meteor.isClient
+  Template.profileEdit.onCreated ->
+    @subscribe('userProfile')
+
+  Template.profileEdit.helpers
+    userProfile: ->
+      UserProfiles.findOne()
+
+  Template.profileEdit.events
+    'submit form': (event) ->
+      event.preventDefault()
+      form = event.target
+      fields = {
+        fullName: form.fullName?.value
+        jobTitle: form.jobTitle?.value
+        bio: form.bio?.value
+        emailHidden: form.emailHidden?.checked
+      }
+      Meteor.call 'updateProfile', fields, (error, response) ->
+        if error
+          toaster.error("Error")
```

Comment:
It could catch errors due to unexpected server issues even if there's no user input that can cause the error - and I think we'd want the user to at least know their update failed, even if they can't fix it.